admiral/pkg/controller/common/types.go

```diff
@@ -64,6 +64,7 @@ type LabelSet struct {
 	AdmiralIgnoreLabel                  string
 	WorkloadIdentityKey                 string //Should always be used for both label and annotation (using label as the primary, and falling back to annotation if the label is not found)
 	GlobalTrafficDeploymentLabel        string //label used to tie together deployments and globaltrafficpolicy objects. Configured separately from the identity key because this one _must_ be a label
+	EnvKey								string //should be used for annotation, annotation is primary and label (env) as a fallback
```

Comment:
Please update this comment to also specify that we check annotation, then label, then env label